docs/diagnostics/PX1023.md

```diff
@@ -0,0 +1,49 @@
+# PX1023
+This document describes the PX1023 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                               | Type  | Code Fix  | 
+| ------ | --------------------------------------------------------------- | ----- | --------- | 
+| PX1023 | The DAC property is marked with multiple field type attributes. | Error | Available |
+
+## Diagnostic Description
+The DAC property field must have only one field type attribute, such as `PXDBBool` or `PXInt`. 
```

Comment:
@k-s-popova This diagnostic actually also does a few more things. It checks that there are no multiple special attributes like PXDBCalced/PXDBScalar declared on the DAC property. It has code fix that leaves the selected attribute and removes all other special attributes.
Also It check the declared attributes and see if they are derived from the aggregators. If they are it collects info about attributes declared on aggregators and see if the declaration is correct, That means that there aren't multiple incompatible field/special type attributes declared on aggregators in DAC property attribute's  inheritance chain. 
By incompatible field type attributes are understood type attributes which correspond to different simple types. 
For example, PXInt and PXDBInt are compatible because they both correspond to int. But PXString and PXInt are not compatible because one corresponds to int and the other corresponds to string.
If there are attributes with incorrectly declared aggregators then the diagnostic shows error about that but doesn't suggests a code fix.

Reply:
@SENya1990, fixed, please verify.

Reply:
@k-s-popova now everything looks fine, thanks